testing/scripts/test_operator_updates.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+
+from seldon_e2e_utils import (
+    initial_rest_request,
+    rest_request,
+    retry_run,
+    wait_for_status,
+    wait_for_rollout,
+)
+
+
+def assert_model(sdep_name, namespace, initial=False):
+    _request = initial_rest_request if initial else rest_request
+    r = _request(sdep_name, namespace)
+
+    assert r is not None
+    assert r.status_code == 200
+    assert r.json()["data"]["tensor"]["values"] == [1.0, 2.0, 3.0, 4.0]
+
+    retry_run(f"kubectl get -n {namespace} sdep {sdep_name}")
```

Comment:
Why is this commadn run once the model assertions have passed?

Reply:
That is to make sure that the `SeldonDeployment` can be fetched from Kubernetes. That was one of the problems we saw when upgrading from `v0.4.1` to `v0.5.1` and it can happen if something disappears between versions (e.g. the CRD version).

The `retry_run()` func should implicitly assert the exit status code of the command but that's not very clear from the test code. I will make the assertion more explicit there.